www/templates/intro/study-text.html

```diff
@@ -0,0 +1,20 @@
+<ion-content class="has-footer">
+	<!-- <div class="intro-space"></div> -->
+	<div class="intro-title">
+	<center><ion-icon></center>
+	</div>
+	<!-- <div class="intro-space"></div> -->
+	<div class="intro-text">
+	<center><img src="img/stop.png"/ style="width: 80%; height: auto;"></center>
+	<div class="intro-space"></div>
+	<center><b>If you are joining a study, please return to your study instructions and proceed with
+	the second step.</b></center>
```

Comment:
Maybe "and complete the second step before proceeding"?
I know I said that not to include the text about "have completed the second step", but I feel like this small change may sound more like the other step is not to download a new UI but to do something else (maybe survey!) and then come back to the same UI.
I am kind of paranoid about being rejected from app store review...